mapie/tests/test_classification.py

```diff
@@ -23,6 +23,8 @@
 
 from mapie._typing import ArrayLike, NDArray
 from mapie.classification import MapieClassifier
+from mapie.conformity_scores import APS, RAPS
+from mapie.conformity_scores.sets.utils import check_proba_normalized
```

Comment:
We must test that we have exacltly the same result when method is specified and when conformity score is specified (for all conformity scores)

Reply:
I'm also wondering if for future the future, we should write all our tests with conformity_score rather than with method

Reply:
Done 👍 (see `test_same_result_with_score_and_method`)